contrib/generate_profile.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+fdbdir=`pwd`/..
+export LD_LIBRARY_PATH=$fdbdir/lib:$LD_LIBRARY_PATH
+export FDB_CLUSTER_FILE=$fdbdir/fdb.cluster
+export LLVM_PROFILE_FILE=$fdbdir/sandbox/fdb-%p.profraw
+../bin/fdbmonitor --conffile ../sandbox/foundationdb.conf --lockfile ../sandbox/fdbmonitor.pid &
+export LLVM_PROFILE_FILE=$fdbdir/sandbox/cli-%m.profraw
+../bin/fdbcli -C ../fdb.cluster --exec 'configure new ssd single'
+export LLVM_PROFILE_FILE=$fdbdir/sandbox/mako-build-%m.profraw
```

Comment:
Nice to add a comment saying this profile will be ignored

Reply:
after test, I found it's not necessary to ignore build mode